editor/app/ems/[incident_number]/page.tsx

```diff
@@ -77,12 +85,53 @@ export default function EMSDetailsPage({
   }, [ems_pcrs]);
 
   /**
-   * Get all people records linked to crashes associated with these incidents
+   * Hook that gets the 12 hour timestamp interval
+   * to be used for fetching people list for unmatched EMS records
+   */
+  const unmatchedTimeInterval: string[] = useMemo(() => {
+    if (incident?.incident_received_datetime) {
+      // Return time interval only if we have any unmatched ems pcrs
+      if (ems_pcrs?.some((ems) => ems.crash_match_status === "unmatched")) {
+        const incidentTimestamp = parseISO(incident.incident_received_datetime);
+        const time12HoursBefore = subHours(incidentTimestamp, 12).toISOString();
+        const time12HoursAfter = addHours(incidentTimestamp, 12).toISOString();
+        return [time12HoursBefore, time12HoursAfter];
+      }
+    }
+    return [];
+  }, [incident, ems_pcrs]);
+
+  /**
+   * Get all crash records that occurred within 12 hours of the incidents
+   * if any of the ems pcrs have a crash match status of unmatched
+   */
+  const { data: unmatchedCrashes } = useQuery<Crash>({
+    query: unmatchedTimeInterval[0] ? GET_UNMATCHED_EMS_CRASHES : null,
+    variables: {
+      time12HoursBefore: unmatchedTimeInterval[0],
+      time12HoursAfter: unmatchedTimeInterval[1],
+    },
+    typename: "crashes",
+  });
+
+  const unmatchedCrashPks: number[] = unmatchedCrashes
+    ? unmatchedCrashes?.map((crash) => crash.id)
+    : [];
+
+  // Combine and dedupe related crash and unmatched crash pks
+  const allCrashPks = Array.from(
+    new Set([...relatedCrashPks, ...unmatchedCrashPks])
+  );
+
+  /**
+   * Get all people records linked to crashes that were either automatically
+   * matched with the ems record or that occurred within 12 hours of the incident
+   * if it has a crash status of unmatched
    */
   const { data: matchingPeople } = useQuery<PeopleListRow>({
-    query: relatedCrashPks ? GET_MATCHING_PEOPLE : null,
+    query: allCrashPks[0] ? GET_MATCHING_PEOPLE : null,
     variables: {
-      crash_pks: relatedCrashPks,
+      crash_pks: unmatchedTimeInterval[0] ? allCrashPks : relatedCrashPks,
```

Comment:
Ok cool now I see how the results are being controlled in the different steps of the matching 👍